filter/etc/e-smith/templates/etc/postfix/main.cf/01template_vars_rspamd

```diff
@@ -1,7 +1,11 @@
 {
-    # Smtpd_milters
-    # Milter used by rspamd
     #
-    push @smtpd_milters, 'unix:/var/run/rspamd/worker-proxy' if $rspamd{'status'} eq 'enabled';
+    # 01template_vars_rspamd -- prepend rspamd filter checks
+    #
+    if($rspamd{'status'} eq 'enabled') {
+        unshift @smtpd_milters, 'unix:/var/run/rspamd/worker-proxy' ;
```

Comment:
`unshift` prepends the rspamd checks before opendkim: should not be necessary, as rspamd does not alter the message but...